src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  gap: 1px;
+  width: $day-size * $days-in-week + 20px;
+  height: 100vh;
+  padding: 10px;
+
+  &__cell {
+    box-sizing: border-box;
+    width: $day-size;
+    height: $day-size;
+    position:relative;
+    transition-duration: 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+
+   @each $name, $margin-left in $weekday {
+    &--start-day--#{$name} {
+      .calendar__cell:first-child {
```

Comment:
Let's do the same but without nested selectors

Reply:
@Olexandr6 with modifier? dont have other option, sorry